src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -1816,6 +1816,9 @@ CHIP_ERROR ExtractVIDPIDFromX509Cert(const ByteSpan & certificate, AttestationCe
     }
 
 exit:
+    ASN1_OBJECT_free(commonNameObj);
```

Comment:
@emargolis @tcarmelveilleux - this seemed a leak on object allocation from text. Changed to make asan tests pass for me, but unsure why CI never saw these,